docs/src/main/sphinx/functions/aggregate.md

```diff
@@ -241,7 +241,24 @@ results in:
  200 | b,c
 ```
 
-The current implementation of `LISTAGG` function does not support window frames.
+This aggregation function can be also used with `FILTER` keyword:
+
+```
+SELECT listagg(value, ',') WITHIN GROUP (ORDER BY id) FILTER (WHERE id % 2 = 0) csv_value
+FROM (
+     VALUES (1, 'a'), (2, 'b'), (3, 'c'), (4, 'd')
+) t(id, value)
+```
+
+results in:
```

Comment:
We should explain what is happening in the query - the filter causes only the elements to get included that pass the where condition .. so where the id is an even number due to the modulo operator and the remainder being 0 ..